pkg/service/external/external_test.go

```diff
@@ -0,0 +1,56 @@
+package external
+
+import (
+	"context"
+	"testing"
+
+	"github.com/micro/go-micro/v2/registry"
+	"github.com/owncloud/ocis-pkg/v2/log"
+)
+
+func TestRegisterGRPCEndpoint(t *testing.T) {
```

Comment:
IMHO this test is not needed, this is an external dependency, you're testing micro.

Reply:
I would like to keep the test as: 
- it tests that the method really registers a service
- it tests the cancellation which removes the service.